test/integration/userStatus.test.js

```diff
@@ -204,7 +237,7 @@ describe("UserStatus", function () {
     it("Should return 200 for deletion by Super User", function (done) {
```

Comment:
can we add one more test for unauthorized i.e without passing any cookie?

Reply:
It's covered in the test case above if the user passes the incorrect cookie it will throw `Unauthorized User`